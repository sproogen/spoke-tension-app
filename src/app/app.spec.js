import React from 'react'
import { shallow } from 'enzyme'
import App from './app'

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
