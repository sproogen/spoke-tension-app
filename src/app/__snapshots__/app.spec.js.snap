// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<SpokeTable
  spokeCount={16}
  spokeId="round-1.8"
  toolId="x-tools"
/>
`;
